From e5a73988f6790fb6961801f0db5992d8bd353e67 Mon Sep 17 00:00:00 2001
From: Ralph Bean <rbean@redhat.com>
Date: Thu, 22 Aug 2013 14:42:58 -0400
Subject: [PATCH] old nose compat

---
 setup.cfg | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 8f6983c..a8b1d1d 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,8 +1,6 @@
 [nosetests]
-logging-clear-handlers = true
 with-coverage = true
 cover-package = urllib3
-cover-min-percentage = 100
 
 [egg_info]
 tag_build = 
-- 
1.8.3.1

